apps/ops-func/src/adapters/functions/split-delete-messages.ts

```diff
@@ -0,0 +1,29 @@
+import { StorageBlobHandler } from "@azure/functions";
+import { QueueClient } from "@azure/storage-queue";
+
+export const splitDeleteMessage =
+  (queueClient: QueueClient): StorageBlobHandler =>
+  async (blob, context) => {
+    if (!Buffer.isBuffer(blob)) {
+      context.error("Invalid input blob file");
+      return;
+    }
+    const lines = blob.toString("utf-8").trim().split("\n");
+
+    lines.forEach((line) => {
+      const [fiscalCode, messageId] = line.split(",");
+
+      if (!fiscalCode || !messageId) {
+        context.error(
+          `Invalid pair fiscalCode: ${fiscalCode}, messageId: ${messageId}`,
+        );
+        return;
+      }
+
+      return queueClient.sendMessage(
+        Buffer.from(JSON.stringify({ fiscalCode, messageId })).toString(
+          "base64",
+        ),
```

Comment:
this can be simplified using `queue output` (removing the need to use a custom queue logic)

Reply:
[55869a0](https://github.com/pagopa/io-messages/pull/181/commits/55869a0249f8786161a8f1526966c7406e03865b)